week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,5 +1,54 @@
-function setAlarm() {}
+function secondsToHms(d) {
+  d = Number(d);
```

Comment:
is this to convert from a string?  
It looks like you're already passing in a number.